docs/user-guide/styling-your-docs.md

```diff
@@ -119,17 +117,17 @@ h1 {
 }
 ```
 
-!!! Warning
+!!! note
```

Comment:
This seemed more appropriate as a "note" than a "warning" to me. If you disagree, I can change it back. Was not actually raised by the linter.